test/helpers.js

```diff
@@ -8,7 +8,7 @@ var usersRef = firebaseRef.child("users");
 var users = [
   {email: "mrgrossm@umich.edu", password: "m", id: "", firstName: "Matt", lastName: "Gross Man"},
   {email: "nickmorg@umich.edu", password: "n", id: "", firstName: "Nick", lastName: "Mo"},
-  {email: "kenzshelley@gmail.com", password: "p", id: "", firstName: "Kenz", lastName: "Shelley"}
+  {email: "kenzshelley@gmail.com", password: "p", id: "", firstName: "Kenz", lastName: "Smelley"}
```

Comment:
I like to think that you were teased in elementary school with this name and it has permanently scarred you.